TypeCobol.Grammar/Grammars/Cobol/CobolWords.g4

```diff
@@ -727,8 +773,7 @@ superObjectIdentifier: SUPER;
 // EBCDIC
 // Specifies the EBCDIC character set.
 
-standardCollatingSequence: STANDARD_1 | STANDARD_2 | NATIVE | EBCDIC;
```

Comment:
Are you sure that all grammar rules that reference `standardCollatingSequence` don't need EBCDIC ?